sonar-python-plugin/src/main/java/org/sonar/plugins/python/indexer/PythonIndexer.java

```diff
@@ -89,6 +89,10 @@ public boolean canBeScannedWithoutParsing(InputFile inputFile) {
     return false;
   }
 
+  public boolean canBeFullyScannedWithoutParsing(InputFile inputFile) {
+    return false;
+  }
```

Comment:
Also, I think we should add a comment distinguishing the two cases:
- The file is unchanged and does not depend on a changed file -> Python checks are not executed, UCFGs and IR are retrieved from the cache.
- The file is unchanged but depends on a changed file -> Python checks are not exceuted, but UCFGS and IRs need to be regenerated.